src/dstack/_internal/server/services/runs.py

```diff
@@ -89,7 +89,7 @@
     BackendType.DATACRUNCH,
     BackendType.GCP,
     BackendType.LAMBDA,
-    BackendType.TENSORDOCK,
+    BackendType.TENSORDOCK
```

Comment:
It need a comma here
```suggestion
    BackendType.TENSORDOCK,
```